Modules/posixmodule.c

```diff
@@ -16699,6 +16968,10 @@ static const struct have_function {
     {"HAVE_EVENTFD", NULL},
 #endif
 
+#ifdef HAVE_TIMERFD_API
+    {"HAVE_TIMERFD_API", NULL},
```

Comment:
Is it really useful? Why not just checking for hasattr(os, "timerfd_create")? I suggest removing it.

Reply:
@vstinner 
Could you suggest a concrete code?
Or you can push it directly.

Reply:
I suggest to remove this constant. That's my suggestion.

Reply:
@vstinner 
Only Linux suppports timerfd API.
Is it OK to just remove it?

Reply:
@vstinner 
timerfd API has timerfd_create, and timerfd_setttime, timerfd_gettime.
So I create the macro.

Replaced it with HAVE_TIMERFD_CREATE.
Do this resolve your concerns?